README.md

```diff
@@ -1,7 +1,331 @@
 # Coveo C# Platform SDK
-The offical Coveo C# SDK allows your software to communicate with the Coveo Cloud Platform public APIs and the Push API.
+[![NuGet version (Coveo.Connectors.Utilities.PlatformSdk)](https://img.shields.io/nuget/v/Coveo.Connectors.Utilities.PlatformSdk.svg?style=flat-square)](https://www.nuget.org/packages/Coveo.Connectors.Utilities.PlatformSdk/)
 
-The code of the SDK is not open-source. This repo is meant to provide feedback, bug reporting, etc.
+The official Coveo C# SDK allows your software to communicate with the Coveo Cloud Platform public APIs and the Push API.
+
+The package is .NET Standard 2.0 compatible thus works for .NET Core and .NET Framework
```

Comment:
"thus" tout seul sonne un peu bizarre ici, je dirais plus "and thus works with" ou "and therefore works with". s à frameworks aussi.

Reply:
"and therefore works with" it is!

Pas de "s" a framework, framework ici fait partie du vrai nom du vieux dotnet